passes/techmap/libparse.cc

```diff
@@ -236,10 +236,13 @@ LibertyAst *LibertyParser::parse()
 
 		if (tok == ':' && ast->value.empty()) {
 			tok = lexer(ast->value);
-			if (tok != 'v')
-				error();
-			tok = lexer(str);
-			while (tok == '+' || tok == '-' || tok == '*' || tok == '/') {
+			if (tok != 'v') {
+			  //Synopsys-style unquoted identifiers issue#3498
+			} else {
+			   //Liberty canonical identifier including double quotes
+			  tok = lexer(str);
+			}			
```

Comment:
This probably should be inverted to prevent the empty branch:

```suggestion
			if (tok == 'v') {
    			tok = lexer(str);
			}
```